cmd/generate-key.go

```diff
@@ -23,9 +23,9 @@ var fileName string
 
 var generateKeyCmd = &cobra.Command{
 	Use:   "generate-key <directory>",
```

Comment:
why is the directory in <> ?

Reply:
You must ask yourself that.  Looks like I didn't change that line.  It isn't a directory really right?